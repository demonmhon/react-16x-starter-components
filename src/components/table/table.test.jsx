import React from 'react';
import { create } from 'react-test-renderer';
import Table from './table';

describe('Table component', () => {
  test('it matches the snapshot', () => {
    const component = create(<Table />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
