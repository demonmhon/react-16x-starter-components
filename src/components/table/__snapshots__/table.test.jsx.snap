// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Table component it matches the snapshot 1`] = `
<table
  className="table"
>
  <colgroup />
  <thead>
    <tr />
  </thead>
  <tbody />
</table>
`;
